portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,11 @@
   <script src="script.js"></script>
 </head>
 
-<body onload="getComments()">
+<body onload="getComments();checkLoginStatus();">
```

Comment:
onload should be clean and only have one thing. Maybe create a new method `loadPage` or something like that calls all the methods needed to load more stuff in the page. Think of it like a `main` method.

Reply:
Done.

Actually I thought about that and then "nah" -- maybe from now on I should re-calibrate. When I find something ugly but too lazy to change, I should refactor the code.